### ISSUE TITLE
SPIKE: Research how to set up build pipelines in GitHub

### ISSUE THREAD

**Issue by acrodemocide** [acrodemocide/StockAnalyzerApi#18]
We need to research how to set up build pipelines in GitHub so that we can get the proper build pipelines set up for web.portfolio-analyzer and StockAnalyzerAPI. Ultimately, we want build pipelines created for both projects such that we are able to ensure that we get a successful build. Part of the build pipelines is to also provide some automated safety measures that must pass before we can merge into master.

**Acceptance Criteria**
1- We have two work items created -- one for creating build pipelines for web.portfolio-analyzer and one for StockAnalyzerAPI both filled with details to easily identify what needs to be done and which can be estimated properly.

OR

2- We have completed the above 2 work items if we find that after doing research that the work to actually implement the pipeline adds little or no more effort to this spike.
